### PR TITLE
Make changing of twitter handler simpler

### PR DESCRIPTION
**Pull request by bobek**
I would like to use Tilex for us as well. To do so I have started to tweak configuration, so it is easier to change things like twitter handler. I am not 100% sure, how you want to approach this though.

Totally unrelated note - before you upgrade Ecto be aware of https://github.com/elixir-ecto/ecto/issues/2217

### CODE REVIEW

**Comment by bobek** [comment]
Thank you @tmock12 . I totally see you point. I cannot say that I am 100% happy with those. I will submit some other pull requests, but it seems that some level of local repo changes will be necessary (e.g.  do I really want to move "about" to configuration etc).

File: config/config.exs [hunk 1]
```diff
@@ -18,6 +18,18 @@ config :tilex, TilexWeb.Endpoint,
            adapter: Phoenix.PubSub.PG2],
   http: [protocol_options: [max_request_line_length: 8192, max_header_value_length: 8192]]
 
+# Provide reasonable default for configuration options
+config :tilex, :page_size, 5
+config :tilex, :auth_controller, AuthController
+config :tilex, :slack_notifier, Tilex.Slack
+config :tilex, :twitter_notifier, Tilex.Twitter
+config :tilex, :organization_name, System.get_env("ORGANIZATION_NAME")
+config :tilex, :ga_identifier, System.get_env("GA_IDENTIFIER")
```
Comment (jwworth) [inline config/config.exs]: The changes to this file are great. The only thing I'd like to see is the `GA_IDENTIFIER`moved back to the production config file. We load the JS only if this variable is set, and we only want Google Analytics in production.

Setting it in the default config has led us to accidentally run Google Analytics in dev and test before, as @tmock12 noted.

**Comment by bobek** [comment]
Sorry guyz, I was a bit snowed under other stuff. I've tried to made changes to address your comments. Let me know what do you think.

**Comment by jwworth** [comment]
Thank you! 👍

**Pull request merged**